content/blog/2023/04/04/conscious-community-practices.md

```diff
@@ -5,7 +5,7 @@ categories:
   - commmunity-call
 authors: 
   - nathen-fitchen
-image: /assets/images/valerie-community-call.svg
```

Comment:
what is the the problem with svg?

Reply:
ok, i get i. these are 15mb. We need to explain to folks that svg are only for *vector*! 🤦‍♂️